lib/formats/sh.c.in

```diff
@@ -326,13 +326,15 @@ static void rm_fmt_head(RmSession *session, RmFmtHandler *parent, FILE *out) {
     char *equal_extra_args = rm_fmt_sh_get_extra_equal_args(session);
 
     /* Fill in all placeholders in the script template */
+    /* This is a brittle hack of which the author should be ashamed. */
```

Comment:
The author is ashamed. But try to do actual templating in C without dependencies. :stuck_out_tongue: